lib/scavenger_hunt/app/assets/stylesheets/scavenger_hunt/components/_helper.sass

```diff
@@ -7,8 +7,6 @@ a.no-border
   font-weight: bold
   text-transform: uppercase
 
-.hint-text
-  font-size: $font-size-h1
-  font-weight: 600
-  line-height: $line-height-h1
-  margin: 0 0 $superview 0
+.end-game
+  color: #000
+  border-color: #000 !important
```

Comment:
This makes it so that the "END GAME" link on the view is always black as shown in the design. One thing I can't quite figure out is how to get this style to apply when the inverse color styling appears. It doesn't seem to respect this !important. Also, this `!important` exists because the border wouldn't change color because of an `!important` on the border color that applies to the rest of the links in Scavenger Hunt.

Reply:
One more thought: please avoid hard-coding colors into the various components. Instead, use the app's global theme, e.g. these should be `$text` and not `#000`.

Reply:
@flipsasser That makes total sense! So, this isn't a thing here, but I'm curious to know incase I ever come across this in the future: if we need to add one new color and it is only being applied in one component, would we create a new global variable or just place that one color hardcoded in the component?